lightwood/analysis/nc/calibrate.py

```diff
@@ -363,19 +371,34 @@ def explain(self, row_insights: pd.DataFrame, global_insights: Dict[str, object]
             if ns.tss.is_timeseries and ns.tss.nr_predictions > 1 and is_numerical:
                 row_insights = add_tn_conf_bounds(row_insights, ns.tss)
 
+            # clip bounds if necessary
+            if is_numerical:
+                lower_limit = 0.0 if ns.positive_domain else -pow(2, 62)
+                upper_limit = pow(2, 62)
```

Comment:
phew, 2**62 a safe upper bound?

Reply:
question -is there a reason we have clipping?

Reply:
Note: this bound was not introduced in this PR. Having said that, it's in place to avoid faulty overflows if a mixer is faulty.